src/gt4sd/frameworks/molgx/requirements.txt

```diff
@@ -0,0 +1,8 @@
+# pypi requirements
```

Comment:
the reqs should be compatible with the gt4sd one and merged there.

Reply:
upper bound are discouraged, also do you really need ipython for the library itself?

Reply:
Yes, MolGX imports ipython.  It is necessary when MolGX is used as a standalone application.

Reply:
A set of packages required by MolGX have been merged into requirements.txt in the top directory.